{"metadata": {"dbt_schema_version": "https://schemas.getdbt.com/dbt/run-results/v2.json", "dbt_version": "0.20.2", "generated_at": "2021-09-27T14:05:23.480152Z", "invocation_id": "a6b74cf6-24b2-4deb-839d-10355b6c971e", "env": {}}, "results": [{"status": "error", "timing": [], "thread_id": "Thread-2", "execution_time": 1.0186655521392822, "adapter_response": {}, "message": "Compilation Error in model station_summary (models/station_summary.sql)\n  When searching for a relation, dbt found an approximate match. Instead of guessing \n  which relation to use, dbt will move on. Please delete \"tech_stack\".\"dbt\".\"station_Summary\", or rename it to be less ambiguous.\n  Searched for: \"tech_stack\".\"dbt\".\"station_summary\"\n  Found: \"tech_stack\".\"dbt\".\"station_Summary\"\n  \n  > in macro materialization_table_default (macros/materializations/table/table.sql)\n  > called by model station_summary (models/station_summary.sql)", "failures": null, "unique_id": "model.my_new_project.station_summary"}, {"status": "error", "timing": [], "thread_id": "Thread-1", "execution_time": 1.1543715000152588, "adapter_response": {}, "message": "Database Error in model station_metadata (models/station_metadata.sql)\n  relation \"analytics.i80stations\" does not exist\n  LINE 9: FROM analytics.I80Stations\n               ^\n  compiled SQL at target/run/my_new_project/models/station_metadata.sql", "failures": null, "unique_id": "model.my_new_project.station_metadata"}, {"status": "skipped", "timing": [], "thread_id": "Thread-4", "execution_time": 0.0, "adapter_response": {}, "message": null, "failures": null, "unique_id": "model.my_new_project.merged_station"}], "elapsed_time": 6.537644624710083, "args": {"log_format": "default", "write_json": true, "use_experimental_parser": false, "profiles_dir": "/home/neba/.dbt", "use_cache": true, "version_check": true, "which": "run", "rpc_method": "run"}}